lib/order.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'send_sms'
+require_relative 'restaurant'
+
+class Order
+
+  attr_reader :menu, :order
+
+  def initialize(restaurant_klass=Restaurant)
+    @restaurant = restaurant_klass
+  end
+
+  def whats_available
+    @restaurant::MENUS.keys
+  end
+
+  def new_order(type=:Italian)
+    reset_order
+    @menu = @restaurant.new(type).menu
+    "A new order has been opened. Please read the menu"
+  end
+
+  def add(item, quantity=1)
+    menu.include?(item) ? adds_with_message(item, quantity) : "Sorry, your selection is unavailable"
```

Comment:
Metrics/LineLength: Line is too long. [100/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)